Post-competition article for SR2024 (#564)
srobo/website: _posts/2024-04-16-mai-win-sr2024.md

---
title: Gymnasium Markt Indersdorf win Student Robotics Competition 2024!
image: /images/content/blog/sr2024/sr2024-photo.jpg
image_alt: All the teams from SR2024

> description: SR2024 came to an exciting conclusion with a victory for Haberdashers' School

Reviewer: ```suggestion
description: SR2024 came to an exciting conclusion with a victory for Gymnasium Markt Indersdorf
```

Author: 67327297b76f5532a9255cb3b37960ddf1ec7448 :facepalm:

---

{% assign photo_full = "/images/content/blog/sr2024/sr2024-photo.large.jpg" | prepend: site.baseurl %}
{% include figure.html
           src="/images/content/blog/sr2024/sr2024-photo.jpg"
           url=photo_full
           caption="The teams and volunteers from SR2024"
           alt="
            The competitors and volunteers from the SR2024 competition, standing in and around the arena.
            At the front the competitors hold a Student Robotics banner, with a selection of robots on the ground in front of them.
            The volunteers are all wearing the customary blue tee-shirts identifying them as such.
          "
%}

> [Gymnasium Markt Indersdorf][MAI] took 1st place in Student Robotics 2024 after an exciting final match against [The Ladies' College Guernsey][TLC], [Headington School][HSO] and [Wisbech Grammar School][WGS]. All the teams have worked amazingly hard over the past six months to design, build & program their robots

Reviewer: Issue: End of sentence. 
```suggestion
[Gymnasium Markt Indersdorf][MAI] took 1st place in Student Robotics 2024 after an exciting final match against [The Ladies' College Guernsey][TLC], [Headington School][HSO] and [Wisbech Grammar School][WGS]. All the teams have worked amazingly hard over the past six months to design, build & program their robots.
```

## The game: Final Frontier

{% include figure.html
           class="right"
           src="/images/content/blog/sr2024/arena.png"
           caption="The 'Final Frontier' arena"
           alt="
            A square arena with each of four 'Planet' scoring zones placed along the edges.
            Those surround an 'Asteroid field' featuring 16 'Asteroids' and in the very center is an 'Egg' on a raised 'Nest'.
            Robots start in the middle of their home Planet, with a 'Spaceship' (also on their Planet) to their right.
          "
%}

> This year’s game, *Final Frontier*, plunged teams into an interstellar battle for dominance among four formidable civilizations, all vying for control of the cosmos. Teams had to retrieve Asteroids scattered throughout the Asteroid field and transport them back to their home Planet or, even better, their intergalactic Spaceship currently under construction.

Reviewer: Nit:
```suggestion
This year’s game, *Final Frontier*, plunged teams into an interstellar battle for dominance between four formidable civilizations, all vying for control of the cosmos. Teams had to retrieve Asteroids scattered throughout the Asteroid field and transport them back to their home Planet or, even better, their intergalactic Spaceship currently under construction.
```

> However the stakes are high as within the confines of the asteroid belt lies The Egg -- a cosmic time bomb that, if disturbed, would spell chaos. Teams had to strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With Asteroids worth points, Spaceships to manage, and The Egg lurking, the universe had never been this competitive.

Reviewer: Nit: Oxford comma
```suggestion
However, the stakes are high, as within the confines of the asteroid belt lies The Egg -- a cosmic time bomb that, if disturbed, would spell chaos. Teams had to strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With Asteroids worth points, Spaceships to manage, and The Egg lurking, the universe had never been this competitive.
```

Author: Nope.

## Challenges

Following on from last year’s success, this year's competition offered teams a set of Challenges to be completed throughout the competition year. These encouraged teams to start work on their robots early and covered movement, mechanics, and vision. They could be approached in any order, and completing challenges before certain deadlines earned the teams bonus league points.

## Virtual Competition

> The first part of our league this year was a virtual competition, broadcast live in March. Across twenty-one matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. Wisbech Grammar School offered the best performance on the day, coming 1st in all four of their matches, though the team from Haberdashers’ Schools remained at the top of the leaderboard due to points they'd earned from the Challenges.

Reviewer: Suggestion: Might be nice to link to the virtual competition page here, mostly as it contains the livestream.

```suggestion
The first part of our league this year was a [virtual competition]({{ '/events/sr2024/virtual-competition/' | prepend: site.baseurl }}), broadcast live in March. Across twenty-one matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. Wisbech Grammar School offered the best performance on the day, coming 1st in all four of their matches, though the team from Haberdashers’ Schools remained at the top of the leaderboard due to points they'd earned from the Challenges.
```

## League Matches

During the in person event teams participated in a further 69 league matches, allowing the robots to improve and evolve both their construction and strategies. The robots displayed impressive agility, precision and ingenuity as they navigated the arena to capture Asteroids (and Spaceships in some cases) as well as outmanoeuvre their opponents.

> At the end of the league there were only two league points separating the top three teams (The Ladies' College Guernsey, Wisbech Grammar School and [Royal Grammar School Guildford][RGS]), with the next few teams not far behind.

Reviewer: ```suggestion
At the end of the league there were only two league points separating the top three teams ([The Ladies' College Guernsey][TLC], [Wisbech Grammar School][WGS] and [Royal Grammar School Guildford][RGS]), with the next few teams not far behind.
```

Author: I'd been following the wikipedia rule of not linking things already linked to.

Author: ec4778f

In a surprise move at the start of the match Wisbech Grammar School immediately captured The Ladies’ College's Spaceship. This appeared to confuse The Ladies’ College's robot, though it soon switched to placing Asteroids on its Planet instead. Meanwhile Gymnasium Markt Indersdorf managed to get a token into their own Spaceship, giving them a commanding lead.

The Ladies’ College and Headington School each captured two Asteroids, however Headington School's robot gifted The Ladies’ College a third in the late stages of the match. Wisbech Grammar School's consistent performance let them capture four Asteroids, and might have been enough to win had Gymnasium Markt Indersdorf not also captured some Asteroids into its robot.

Wisbech Grammar School were the source of another tense moment near the very end of the match when they attempted to steal The Ladies’ College's first Asteroid, however they were unable to navigate off the Planet in time meaning that the Asteroid still scored for The Ladies’ College.

## Results

> - Winner: MAI – Gymnasium Markt Indersdorf

Reviewer: Suggestion: As this is the winners table, I think we should explicitly link out for every team, not just some of them.

Author: ec4778f
